static/index.html

```diff
@@ -4,7 +4,13 @@
         <meta charset="utf-8">
         <meta content="width=device-width, initial-scale=1" name="viewport">
         <title>ankerctl</title>
-        <link href="/static/vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css" rel="stylesheet">
+        <link rel="stylesheet" href="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css') }}">
+        <link rel="stylesheet" href="{{ url_for('static', filename='vendor/bootstrap-icons-1.10.5/font/bootstrap-icons.min.css') }}">
+		<link rel="stylesheet" href="{{ url_for('static', filename='ankersrv.css') }}">
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/js/bootstrap.bundle.min.js')}}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/cash.min.js') }}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='vendor/jmuxer.min.js') }}"></script>
+        <script type="text/javascript" src="{{ url_for('static', filename='ankersrv.js') }}"></script>
```

Comment:
Why were all `<script>` tags moved to `<head>`?

Having them in `<head>` requires all scripts to be downloaded before the browser can start rendering. It's almost never what we want.